src/main/scala/intent/sbt/Runner.scala

```diff
@@ -83,12 +83,12 @@ class SbtTask(td: TaskDef, suit: IntentStructure, runner: TestSuiteRunner, focus
   override def taskDef = td
 
   // Tagging tests or suites are currently not supported
-  override def tags(): Array[String] = Array.empty
+  override def tags(): Array[String | Null] = Array.empty
 
-  override def execute(handler: EventHandler, loggers: Array[Logger]): Array[Task] =
+  override def execute(handler: EventHandler | Null, loggers: Array[Logger | Null] | Null): Array[Task | Null] =
     implicit val ec = ExecutionContext.global
     // Hmm, do we need really to block here? Does SBT come with something included to be async
-    Await.result(executeSuite(handler, loggers), Duration.Inf)
+    Await.result(executeSuite(handler.nn, loggers.nn.map(_.nn)).asInstanceOf[Future[Array[Task | Null]]], Duration.Inf)
```

Comment:
**Overridden method returns array**
If a method overrides a java method which returns an array, then the return type must be modified from `Array[T]` to `Array[T | Null]`. However, when this return value is then used, we need to cast it back to `Array[T]`.